modules/ai/assets/js/editor/ai-layout-behavior.js

```diff
@@ -24,6 +24,8 @@ export default class AiLayoutBehavior extends Marionette.Behavior {
 	onAiButtonClick( e ) {
 		e.stopPropagation();
 
+		window.elementorAiCurrentContext = this.getOption( 'context' );
```

Comment:
It's overriding the context, maybe you need to restore it after the current action?

Reply:
it gets updated on every AI call, so it will be up to date in the next use